components/FleetRepos.vue

```diff
@@ -89,11 +94,31 @@ export default {
     <template #cell:clustersReady="{row}">
       <span v-if="!row.clusterInfo" class="text-muted">&mdash;</span>
       <span v-else-if="row.clusterInfo.unready" class="text-warning">{{ row.clusterInfo.ready }}/{{ row.clusterInfo.total }}</span>
-      <span v-else>{{ row.clusterInfo.total }}</span>
+      <span v-else class="cluster-count-info">
+        {{ row.clusterInfo.total }}/{{ row.clusterInfo.total }}
```

Comment:
This would need to show `ready / total`

Reply:
completely true! Thanks for catching this! 🙏